app/src/main/java/com/example/parkingsystem/utilities/Picker.java

```diff
@@ -14,8 +15,13 @@
 import static java.util.Calendar.MINUTE;
 
 public class Picker {
+
+    private static final String RESERVATION_FRAGMENT_TAG = "RESERVATION_FRAGMENT";
+    @SuppressLint("SimpleDateFormat")
+    private final SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yy-MM-dd HH:mm");
```

Comment:
You can create this instance just before using it, as it was before. But you should define the string you use to initialize it as a constant.